docker-images/notebook/Dockerfile

```diff
@@ -3,76 +3,81 @@ FROM jupyter/base-notebook:4417b81d04b7
 
 USER root
 RUN apt-get update \
-  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse vim git graphviz \
+  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse vim git \
              man tree build-essential \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 USER $NB_USER
 
-RUN conda install --yes \
-    -c conda-forge \
-    -c defaults \
-    -c pyviz/label/dev \
-    -c bokeh/channel/dev \
-    -c intake \
-    bokeh=0.12.16 \
+RUN conda update conda --yes \
+  && conda config --remove channels defaults
+
+RUN conda install --yes  \
+    appmode \
+    arrow \
+    pyviz/label/earthsim::bokeh \
```

Comment:
`conda-forge` should have the latest `holowviews` and `bokeh` now that the pyviz devs are using `conda-forge` for releases. @philippjfr can probable comment on the differences between the two channels.